contrib/platform/src/com/sun/jna/platform/win32/DBT.java

```diff
@@ -308,6 +309,26 @@ public DEV_BROADCAST_PORT(Pointer memory) {
             super(memory);
             read();
         }
+
+        @Override
+        public void read() {
+            this.dbcp_size = (Integer) this.readField("dbcp_size");
```

Comment:
As a slight performance improvement, rather than using reflection, since we know the size is the first element of the structure, can we not just use `getPointer().getInt(0)`?  (Also elsewhere in other structures.)

Reply:
Valid point - updated the PR with the change, ran it again through manual testing and found it working.